### ISSUE TITLE
 batch editor creates duplicate localities 

### ISSUE THREAD

**Issue by timo11** [specify/specify6#103]
If you batch edit a co query that has 5 cos from the same locality with lat/lng in degrees minutes seconds, and you change the lat/lngs for all 5 cos to the same coords in decimal degrees format, you are likely to end up with 5 new identical localities.

**Comment by timo11** [comment]
ddd

**Comment by timo11** [comment]
If you batch edit a co query that has 5 cos from the same locality with lat/lng in degrees minutes seconds, and you change the lat/lngs for all 5 cos to the same coords in decimal degrees format, you are likely to end up with 5 new identical localities.

**Comment by yvonneekanim** [comment]
I added a new locality with lat/long in degree minute seconds and created 4 collection objects with this locality. I made a co query and batch edited 3 of the locality's lat/long to the same value in decimal degrees. When I did a locality query, I had two localities, on with lat/long in decimal degrees and the other in degrees minutes seconds.

**Comment by garrettgirard** [comment]
I also tried backward translating it: I used 6 COs that had the same locality. The locality was originally in decimal degrees. I changed the format for 2 to degrees decimal seconds and 2 to degrees minutes decimal seconds. This resulted in a total of 3 separate localities, all the same except for coordinate format. I then changed one of each of the new formats back to decimal degrees and this did not create any new localities; it appears to just have re-linked these COs back to the first locality.
